src/libmongocrypt-compat/mongocrypt-export.h

```diff
@@ -0,0 +1,7 @@
+#ifndef MONGOCRYPT_EXPORT_H
+#define MONGOCRYPT_EXPORT_H
+
+#define MONGOCRYPT_EXPORT
+#define MONGOCRYPT_NO_EXPORT
```

Comment:
@alcaeus I assume my suggestion to define these as empty strings worked?

Reply:
Yes, that worked beautifully 👍